README.md

```diff
@@ -1,5 +1,7 @@
 # react-bpmn
 
+![CI](https://github.com/bpmn-io/react-bpmn/workflows/CI/badge.svg)
```

Comment:
Usually we link to the CI pipeline, too. Could we do that here, as well?

Reply:
Fixed via https://github.com/bpmn-io/react-bpmn/pull/12/commits/752f75392ae21b67eb9c54ba4b506ce991bac3b9.